pkg/appconsts/appconsts.go

```diff
@@ -8,23 +8,25 @@ import (
 	"github.com/tendermint/tendermint/pkg/consts"
 )
 
-// These constants are sourced from:
+// These constants were originally sourced from:
```

Comment:
[Non Blocking][Suggestion]
Probably would make sense to put where this was decided. Something like:
```
// And, then, updated following the discussion under:
// https://github.com/celestiaorg/celestia-app/pull/660#discussion_r958603307
```

Reply:
Lines 11-12 link to celestia-specs because its relevant for several consts in the block below.

I think a `README.md` in the `share/` package that explains the difference between `CompactShare` vs `SparseShare` may be better than a link to the motivating GH discussion but we can include the link you suggest immediately above compact and sparse share consts and then remove once we have such a README.

Reply:
Resolved in https://github.com/celestiaorg/celestia-app/pull/710/commits/a81d232dacfc7610b53a564a6a56c84927753c23

Reply:
Sounds good, thanks